scripts/update-docs.sh

```diff
@@ -22,36 +22,27 @@ do
 
     # Generate latest version of docs
     echo "Generating docs."
-    pushd .
+    pushd . # We're in the root folder
     cd "packages/$SCOPE"
     yarn docs
     cd docs
     gitbook build
-    popd
-
-    # Push docs to parity-js, on gh-pages branch
-    echo "Cloning doc repo."
-    PROJECT_DIR=`pwd`
-    REMOTE_REPO="https://git:$GH_TOKEN@github.com/parity-js/$SCOPE.git"
-    cd /tmp # Clone that repo in the /tmp folder
-    git clone -b gh-pages $REMOTE_REPO new-$SCOPE-docs
-    cd new-$SCOPE-docs
-    cp -r "$PROJECT_DIR/packages/$SCOPE/docs/_book/." .
-    git add .
-
-    echo "Pushing to doc repo."
-    set +e # Don't catch errors in the next lines
-    git commit -m "Update docs"
-    git push
-    set -e
-    cd ..
-    rm -rf new-$SCOPE-docs
-
-    # Go back to root
-    cd $PROJECT_DIR
+
+    # Copy these docs temporarily in a temp folder
+    TMPDIR=$(mktemp -d)
+    cp -r "_book/." $TMPDIR
+    popd # Go back to root folder
+
+    # Copy these docs back to the gh-pages branch
+    git reset --hard HEAD
+    git checkout gh-pages
+    rm -rf $SCOPE
+    cp -r $TMPDIR $SCOPE
+
+    # Return to original branch
```

Comment:
probably need to `commit --allow-empty` and then `commit --amend` in each iteration to prevent `error: Your local changes to the following files would be overwritten by checkout`

Reply:
or alternatively generate docs for all packages in SCOPES and then checkout to gh-pages, copy and commit